dataprofiler/profilers/profiler_options.py

```diff
@@ -869,9 +869,13 @@ class CategoricalOptions(BaseInspectorOptions):
     def __init__(
         self,
         is_enabled: bool = True,
-        top_k_categories: int = None,
+        top_k_categories: int | None = None,
         max_sample_size_to_check_stop_condition: int | None = None,
         stop_condition_unique_value_ratio: float | None = None,
+        cms: bool = False,
+        cms_confidence: float | None = 0.95,
+        cms_relative_error: float | None = 0.01,
+        heavy_hitters_threshold: int | None = None,
```

Comment:
i'm okay with a default of `5000` since that's when we start limiting our dataset.